frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -952,7 +952,12 @@ function FeatureFlagRollout({ readOnly }: { readOnly?: boolean }): JSX.Element {
                             </div>
                             <div className="col-span-4 flex items-center gap-1">
                                 <span>Rollout</span>
-                                <LemonButton onClick={distributeVariantsEqually}>(Redistribute)</LemonButton>
+                                <LemonButton
+                                    onClick={distributeVariantsEqually}
+                                    tooltip="Redistribute variant rollout percentages equally"
+                                >
+                                    <IconBalance />
+                                </LemonButton>
```

Comment:
looks like this
<img width="488" alt="image" src="https://github.com/user-attachments/assets/e6c575b5-eef2-401e-b719-2c09bf399f4c">

Reply:
If I'm gonna nitpick I'd still bump it over to line up with the right side of the column, but if not then I think this is fine 👍

Reply:
nitpick addressed!